mrt/forms/meetings/participant.py

```diff
@@ -207,7 +207,8 @@ def __init__(self, *args, **kwargs):
         super(FlagForm, self).__init__(*args, **kwargs)
         flags = g.meeting.custom_fields.filter_by(
             field_type=CustomField.CHECKBOX,
-            is_primary=True)
+            is_primary=True,
+            custom_field_type=kwargs['custom_field_type'])
```

Comment:
I think it's better to check if this key exists in `kwargs`. Maybe adding something like `custom_field_type = kwargs.pop('custom_field_type', Participant.PARTICIPANT)` before the `super()` call (line 207).